.github/workflows/ci.yml

```diff
@@ -22,4 +22,19 @@ jobs:
       - run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
       - run: cargo build --verbose
       - run: cargo test --verbose --workspace
-  
\ No newline at end of file
+
+  cargo-fmt:
+    name: Cargo fmt
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Fetch Repository
+        uses: actions/checkout@v5
```

Comment:
nit, this is not consistent with the other job, we're using v5 vs v4, and add a name. I think we should use [v5](https://github.com/actions/checkout/blob/main/CHANGELOG.md#v500), I'm not opinionated for the name.